src/app/components/navigation/SharedToolbar.tsx

```diff
@@ -109,6 +140,19 @@ export const SharedToolbar = (props: SharedToolbarProps): JSX.Element => {
                 }}
                 {...other}
             >
+                {getIsFireworkHoliday() && (
+                    <div
+                        id="fireworks"
+                        style={{
+                            position: 'fixed',
```

Comment:
Absolute will keep it tied to the AppBar
```suggestion
                            position: 'absolute',
```